app/api/CardanoClientApi.js

```diff
@@ -307,7 +307,7 @@ export default class CardanoClientApi {
   }
 
   async createAddress(request: CreateAddressRequest) {
-    Logger.debug('CardanoClientApi::createAddress called: ' + stringifyData(request));
+    Logger.debug('CardanoClientApi::createAddress called');
```

Comment:
Why are we not logging the request here anymore?

Reply:
Because createAddress can contain the passphrase! ;)

Reply:
makes sense 😉 merging now!